src/Compilers/Core/CodeAnalysisTest/Text/TextChangeTests.cs

```diff
@@ -10,11 +10,20 @@
 using Roslyn.Test.Utilities;
 using Xunit;
 using System.Collections.Generic;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Xunit.Abstractions;
 
 namespace Microsoft.CodeAnalysis.UnitTests
 {
     public class TextChangeTests
     {
+        private readonly ITestOutputHelper _output;
+
+        public TextChangeTests(ITestOutputHelper output)
```

Comment:
Had not seen this before, this is quite useful.

Reply:
We use it our parsing tests too. That gives a nice copy&paste area (Ctrl-A selects the whole output).

---
In reply to: [501904239](https://github.com/dotnet/roslyn/pull/48420#discussion_r501904239) [](ancestors = 501904239)